### PR TITLE
199 visioncasting section

### PR DESCRIPTION
**Pull request by tamarayustian**
this is the theme section on figma

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   },
   "dependencies": {
     "react-google-recaptcha": "^2.1.0",
+    "react-router-hash-link": "^2.4.3",
```
Comment (albertparedandan) [inline package.json:9]: this is installed in the wrong directory. should be in /ui. right now it's in our project root

File: ui/src/components/retreat/about/AboutContainer.js [hunk 2]
```diff
@@ -35,58 +54,36 @@ const AboutContainer = () => {
...
-              </Button>
+              <NavButton
+                to="/with-everything/about#theme"
+                color="#FFC93E"
+                name="Theme"
+              />
+              <NavButton
+                to="/with-everything/about#speaker"
+                color="#EE794E"
+                name="Speaker Profile"
```
Comment (albertparedandan) [inline ui/src/components/retreat/about/AboutContainer.js:83]: there's an overflow

image

File: ui/src/components/retreat/about/ThemeSection.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                textStyle="inter"
+                fontSize={['12px', '16px', '10px', 'xs', 'sm']}
+              >
+                We believe that God has given us everything that we need, and in
+                response to that, we want to worship Him with everything that we
+                have. We want to believe that as we respond with everything, God
+                is going to do great things in our lives and in our church.
+                <br /> <br />
+                Matthew 13:44 (English Standard Version)
+                <br /> ““The kingdom of heaven is like treasure hidden in a
```
Comment (albertparedandan) [inline ui/src/components/retreat/about/ThemeSection.js:51]: there's an extra "

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: ui/src/components/retreat/about/AboutContainer.js [hunk 4]
```diff
@@ -9,88 +9,78 @@ import {
...
+              <b> With Everything - Church-wide Conference 2022</b>
+            </Text>
+          </Box>
+          <HStack spacing={[2, 5]}>
+            <NavButton
+              to="/with-everything/about#theme"
+              color="#FFC93E"
+              name="Theme"
+            />
+            <NavButton
```
Comment (ksuhartono97) [inline ui/src/components/retreat/about/AboutContainer.js:74]: something seems to be wrong with the speaker profile button it seems to not switch the interface

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

**Review by ksuhartono97 — APPROVED** [review]

File: ui/src/components/retreat/about/AboutContainer.js (hunk 4 above)
Comment (ksuhartono97) [reply ui/src/components/retreat/about/AboutContainer.js:74]: Apparently will be solved with #200

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**